### ISSUE TITLE
Buildozer requirements with opencv issue

### ISSUE THREAD

**Issue by JSVJ** [liyuanrui/kivy-for-android-opencv-demo#4]
I am having a python code which includes CV2 and aruco markers. I installed the OpenCV in python using the command "pip install OpenCV-contrib-python". 

The app works perfectly on the desktop. It also generates the apk file too. However, when I try to open the app it always crashes with the error "ModuleNotFoundError: No module named 'cv2'". 

I tried changing the name of OpenCV in the requirements into OpenCV-python or OpenCV-contrib-python. 
It's not working for any of them.  It returns the error "ImportError: OpenCV loader: missing configuration file: ['config.py']. Check OpenCV installation"

I have attached my bulldozer spec file. 
[buildozer.txt](https://github.com/liyuanrui/kivy-for-android-opencv-demo/files/9672029/buildozer.txt)

**Comment by dxwss** [comment]
I have the same problem. Have you solved it？

**Comment by JSVJ** [comment]
@dxwss Nope. After repeated failures, I was experimenting with different android versions. I was able to achieve in an older android version.
